WeakAurasOptions/RegionOptions/AuraBar.lua

```diff
@@ -85,15 +85,15 @@ local function createOptions(id, data)
       hidden = function() return not WeakAuras.CanHaveTooltip(data) end,
       order = 38
     },
-    symbol_header = {
+    icon_header = {
```

Comment:
This also looks to be unrelated to the rest of the pull request. If/when you squash, please don't accidentally squash this into a commit which says it's talking about dynamic group improvements (or, just also mention in the commit message that you changed this)

Reply:
i tried cherry-picking in another branch, but i'm getting conflicts with master, so i'll make no squash it